packages/react-native-renderer/src/ReactNativeViewConfigRegistry.js

```diff
@@ -44,7 +44,7 @@ export function get(name: string): ReactNativeBaseComponentViewConfig {
     const callback = viewConfigCallbacks.get(name);
     invariant(
       typeof callback === 'function',
-      'View config not found for name %s',
+      'View config not found for %s. If you made a component starting with a lowercase letter, make it uppercase.',
```

Comment:
"made a component" is a bit odd, maybe "wrote"?

Reply:
How about we skip the verb entirely?

"If your component starts with a lowercase letter, make it uppercase."

Reply:
Sounds good